package.json

```diff
@@ -98,6 +98,6 @@
   "snyk": true,
   "dependencies": {
     "@uswds/compile": "1.2.0",
-    "@uswds/uswds": "3.9.0"
```

Comment:
**chore**:  Will need to revert this once testing is finished and feature is accepted.